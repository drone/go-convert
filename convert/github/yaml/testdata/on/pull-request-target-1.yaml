# https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target

on:
  pull_request_target:
    types: [assigned, opened, synchronize, reopened]