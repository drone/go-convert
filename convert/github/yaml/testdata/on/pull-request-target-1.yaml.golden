on:
  pull_request_target:
    types: [assigned, opened, synchronize, reopened]